controllers/snapshot/snapshot_adapter.go

```diff
@@ -72,6 +72,10 @@ func (a *Adapter) EnsureAllIntegrationTestPipelinesExist() (results.OperationRes
 			"Application.Namespace", a.application.Namespace)
 	}
 
+	a.logger.Info("Found IntegrationTestScenarios for application",
```

Comment:
Also good to add this within https://github.com/redhat-appstudio/integration-service/blob/main/helpers/integration.go#L58.

Reply:
Yes, I think it's better to handle the logging here, within the `snapshot_adapter.go`, as we're able to tailor the message to the use-case.